internal/character/natasha/eidolon.go

```diff
@@ -0,0 +1,153 @@
+package natasha
+
+import (
+	"github.com/simimpact/srsim/pkg/engine/event"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	E1                    key.Modifier = "natasha-e1-autoheal"
+	E1PercentThreshold    float64      = 0.30
+	E1HealScale           float64      = 0.15
+	E1HealFlat            float64      = 400
+	E2                    key.Modifier = "natasha-e2"
+	E2ThresholdPercentage float64      = 0.30
+	E2PercentageHOT       float64      = 0.06
+	E2FlatHOT             float64      = 160
+	E4                    key.Modifier = "natasha-e4"
+)
+
+type E1State struct {
+	PercentageThreshold float64
+	HealScale           float64
+	HealFlat            float64
+	alreadyUsed         bool
+}
```

Comment:
* Why use a state here? All of these values appear to be static, and you have defined them as constants. What value is a state providing?
* rather than have an already used state, just remove the E1 modifier after the heal goes off (like what datamine does). This solution is also nice because then in logs the `natasha-e1-autoheal` modifier only shows up while the heal is available